metabase/metabase
# ci(sdk): fix and re-enable e2e workflow for 51 branch

Fixes the e2e workflow for the 51 branch

## How to verify

- E2E was red without the webpack configuration, see this workflow run: https://github.com/metabase/metabase/actions/runs/12352781721/job/34471028626?pr=51327
- E2E is green now, see workflow runs in this commit (which forcibly runs the workflow against this branch): https://github.com/metabase/metabase/pull/51327/commits/19be57ec17575b4de7b533239bf62c0d7bd74ae8
